### ISSUE TITLE
GameGuru MAX - Bug - Material Sounds Too Quiet to Hear

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#3636]
When importing an object into MAX and assigning a material sound, such as Metal, when playing a test game, the sound is too quiet to hear. I had to turn my PC volume up to 80-100% just to hear then sounds and, even then, I could only just hear them. 

1 - Sounds on imported objects need to be much louder.
2 - We need a way to adjust the volume of these sounds on a material basis (some metal surfaces may be quieter or louder, for example)
3 - We need a way to change out these sounds even after import

On the last one, it's easy to forget to assign a sound to a material on an imported model. We need to be able to select the object in the view port, go to the Material Tab, and assign the sound there (or even choose a different one), just like we could on import.

**Comment by davetgc** [comment]
Not sure if this is a bug or a feature request. Seems to be a bit of both.

**Comment by Granada1** [comment]
I think you can call this one a bug , you can’t hear your own footsteps even on metal , a way of adjusting the sound level is needed I think 👍

**Comment by MonkeyFrogStudio** [comment]
@davetgc How can this not be a bug? You cannot hear the footfalls unless you turn up your speakers and PC sound to the max! If you do that, a simple email alert will break your eardrums. ;) It's not that the footfall sounds on imported materials are a little too quiet, it's that they can't be heard at all at normal sound settings and barely heard when the sound is turned all the way up.

**Comment by LeeBamberTGC** [comment]
@MonkeyFrogStudio We had louder footfall sounds in Classic and got complaints about that too :)  It is marked as an enhancement as the current system is working as designed (i.e. very quiet and subtle footfalls). The request is for more control over this, thus a feature request :)

**Comment by LeeBamberTGC** [comment]
NOTE: Ad you can see here, the WAV for footfalls is VERY subtle. INcreasing volume would not necessarily help, we would need the original sound authors to create new footfalls with more meat on the bone:

![image](https://user-images.githubusercontent.com/35840583/186669085-75182ec2-3e15-4755-9be7-75c75e9134e5.png)

**Comment by MonkeyFrogStudio** [comment]
@LeeBamberTGC Here's a video of a level where I am testing out some parts and pieces:

https://user-images.githubusercontent.com/36735507/186670443-c40a7c42-e367-420f-bc4b-4f835b3f04cf.mp4

Tell me, can you hear the footfalls at all with your speakers at a normal or even loud setting? I have to turn my speakers up to 100% to even hear the footfalls and that makes all the other sounds so loud it hurts my ears.

**Comment by MonkeyFrogStudio** [comment]
@LeeBamberTGC Yes, the sounds are there but, like I said, I have to turn up the sound on my system to 100% to even hear them. Do you want to hear what cocking a gun sounds like in comparison? It will blow out your eardrums!

Regardless of the sound issues, I would DEFINITELY like a means of adding my own sound effects with custom materials. In fact, I believe I had requested this before. ;)

Thanks for the compliment on the level test, btw. ;)

**Comment by MonkeyFrogStudio** [comment]
@LeeBamberTGC Compare the sounds of the patrol rifle being reloaded and shot to the footfalls:

https://user-images.githubusercontent.com/36735507/186674313-6b312c6e-88cb-42f6-a22f-4f475e7d1d88.mp4

Make sure to have your volume up loud enough to hear the footfalls and then see how loud the rifle is.

**Comment by synchromesh62** [comment]
Funny thing about footsteps are you can barely hear your own unless you stomp. 
But you can hear other peoples walking around.
Its not my low football sounds that bother me so much as others.

**Comment by MonkeyFrogStudio** [comment]
@synchromesh62 True. We become accustomed to our own sounds (like our heartbeat, etc.), but are more attuned to the sounds other make. However, this is a game we're talking about and, to me, it's part of immersion. I am aware that I make sound when I walk, but can tune it out. I can do the same with a player's footfalls, too. But if they are absent, I notice it very much as it pulls me out of suspension of disbelief. I am not asking for the footfall sounds to be prominent, but to be hearable at normal volumes. If they cannot be heard at normal volumes, why even bother having them at all?

**Comment by GraPhiX-Guru** [comment]
@MonkeyFrogStudio i have sent you a PM on Discord with some sounds to test

**Comment by MonkeyFrogStudio** [comment]
@GraPhiX-Guru Thanks! I've responded on Discord.

**Comment by MonkeyFrogStudio** [comment]
@GraPhiX-Guru That seems much better, thanks!

@LeeBamberTGC Graphix turned up the volume on the metal walking sounds by 25%. That may be a bit too much, actually. But an increase in sound by 15% would probably do it.

**Comment by davetgc** [comment]
That's a good point. Unless walking on gravel or similar.

**Comment by LeeBamberTGC** [comment]
NOTE: In the meantime, here is a way you can increase the volume of footfalls right now. When creating a new project, ensure it is a remote project (so your edits are not erased when an update/verify happens from Steam).  Once created, add (or confirm the existence of) a folder here: "Files\audiobank\materials" and copy ALL the files from the main MAX folder (typically "C:\Program Files (x86)\Steam\steamapps\common\GameGuru MAX\Files\audiobank\materials") into this folder. You can then use the free tool called Audacity to edit the WAV files within each sub-folder to alter the sounds, including their volume.  Audacity lets you do this in batches so its not too tedious to manage this many audio files.

<img width="1470" height="614" alt="Image" src="https://github.com/user-attachments/assets/09e00bc3-8a82-4385-abd5-f2b67e9456e8" />

You can even have fun with the sounds to create your own unique take on them, or even replace them entirely, just make sure the WAV format, general length and profile remains the same to keep the audio in line with the footfall logic in the engine.
